packages/api-proxy/src/middlewares/sdk-injector.ts

```diff
@@ -5,6 +5,9 @@ import { Env, SDK } from "@haechi-labs/henesis-wallet-core";
 @Injectable()
 export class SdkMiddleware implements NestMiddleware {
   use(req: Request, res: Response, next: NextFunction): any {
+    const origin = (req.headers["x-forwarded-for"] ||
+      req.socket.remoteAddress) as string;
```

Comment:
In my opinion, we should keep both x-forwarded-for header and remoteAddress.
If the user who uses API proxy may fill a wrong value for the x-forwarded-for header. We should  collect the both information.